snapcraft/file_utils.py

```diff
@@ -417,3 +418,15 @@ def get_resolved_relative_path(relative_path: str, base_directory: str) -> str:
     filename_relpath = os.path.relpath(filename_abspath, base_directory)
 
     return filename_relpath
+
+
+def _remove_readonly(func, path, excinfo):
+    """Try setting file to writeable if error occurs.
```

Comment:
this docstring does not follow the PEP recommendation we follow https://www.python.org/dev/peps/pep-0257/#one-line-docstrings
Is this a code comment in disguise?

Reply:
fixed (Code comment)